packages/react-wildcat-handoff/src/utils/defaultTemplate.js

```diff
@@ -10,6 +11,7 @@ module.exports = function defaultTemplate(cfg) {
 
     const wildcatConfig = cfg.wildcatConfig;
     const clientSettings = wildcatConfig.clientSettings;
+    const protocol = (((wildcatConfig || {}).serverSettings || {}).appServer || {}).protocol;
```

Comment:
That can't happen, the [default config](https://github.com/nfl/react-wildcat/blob/master/packages/react-wildcat/src/utils/templates/wildcat.config.js) ensures the value is always present. You can go ahead and update the stub instead.